plugin-tester-scala/src/test/scala/example/myapp/helloworld/ErrorReportingSpec.scala

```diff
@@ -65,13 +72,13 @@ class ErrorReportingSpec extends AnyWordSpec with Matchers with ScalaFutures wit
         case Chunked(_, chunks) =>
           chunks.runWith(Sink.last).futureValue match {
             case LastChunk(_, trailingHeaders) => response.headers ++ trailingHeaders
-            case _                             => response.headers
+            case _ => response.headers
```

Comment:
It looks like you have some custom formatter setup in your IDE? Not a big deal.

Reply:
I have standard IntelliJ IDEA formatter, nothing custom